### PR TITLE
add doccano

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute will you be a maintainer?

**Comment by rluria14** [comment]
@conda-forge/staged-recipes this recipe is ready for review

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

File: recipes/doccano/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{% set version = "1.0.4" %}
+{% set node_version = os.environ.get('NODEJS_VERSION') or '13.6.0' %}
```
Comment (xhochy) [inline recipes/doccano/meta.yaml:2]: What is the reason for this very strict pin? Can you put `node 13.*` into `build` and `host` instead?

**Comment by rluria14** [comment]
@xhochy I've tried setting to 'node 13.*' and it wont work. Via issues on the doccano page, it looks like people have had success with 13.6.0 which is the reason why I set it to that version.

File: recipes/doccano/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    - python3 manage.py test server.tests
+
+  source_files:
+    - app/
+
+about:
+  home: https://github.com/doccano/doccano
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (xhochy) [inline recipes/doccano/meta.yaml:94]: You also need to include the licenses of the bundled `npm` packages.

**Comment by xhochy** [comment]
> @xhochy I've tried setting to 'node 13.*' and it wont work. Via issues on the doccano page, it looks like people have had success with 13.6.0 which is the reason why I set it to that version.

Ok, this is going to be a maintenance pain but seems to be the only option for now.

File: recipes/doccano/meta.yaml (hunk 2 above)
Comment (rluria14) [reply recipes/doccano/meta.yaml:37]: What's the preferred method of adding multiple licenses to a recipe?

Comment (chrisburr) [reply recipes/doccano/meta.yaml:37]: Get rid of `license_family`, use a SPDX license expression and provide a list of license files, i.e.
```yaml
  license: MIT AND LGPL-2.0-OR-LATER
  license_file:
    - LICENSE
    - OTHER_LICENSE
```
You can also use things like `MIT AND (LGPL-2.0-or-later OR Apache-2.0) AND BSD-3-Clause`, see https://spdx.org/licenses/

Comment (rluria14) [reply recipes/doccano/meta.yaml:37]: @xhochy After looking at the doccano package json I'm having trouble determining which packages are bundled? Also doesn't look like any of the npm installed packages on conda-forge have multiple licenses in their recipes

Comment (rluria14) [reply recipes/doccano/meta.yaml:37]: @xhochy bump

Comment (xhochy) [reply recipes/doccano/meta.yaml:37]: Passing the review to someone else on that, for `code-server` we actually bundle all licenses but the bundling is done by VSCode: https://github.com/conda-forge/code-server-feedstock/blob/458f7f4d19b0682aaf821177efc210ccbb870169/recipe/meta.yaml#L36

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

**Comment by rluria14** [comment]
@conda-forge-admin bump. please review and merge

**Comment by rluria14** [comment]
@conda-forge/staged-recipes please review

**Comment by beckermr** [comment]
bump @xhochy - you ok to merge this?

File: recipes/doccano/meta.yaml (hunk 2 above)
Comment (beckermr) [reply recipes/doccano/meta.yaml:37]: @conda-forge/core can someone familiar with win take a look at this?

**Pull request merged**